test_scripts/Polices/Policies_Security/ATF_RAI_Assign_Default_Policies_GENEVI.lua

```diff
@@ -0,0 +1,133 @@
+          fileName = "sdl_snapshot.json"
+        },
+        pathToSnaphot
+      )
+    end)
+  EXPECT_NOTIFICATION("OnSystemRequest", {requestType = "PROPRIETARY" })
+  :Do(function(_,data)
+      local CorIdSystemRequest = self.mobileSession:SendRPC ("SystemRequest",
+        {
+          requestType = "PROPRIETARY",
+          fileName = "sdl_snapshot.json"
+        },
+        pathToSnaphot
+      )
+      EXPECT_HMICALL("BasicCommunication.SystemRequest")
+      :Do(function(_,data)
+          self.hmiConnection:SendResponse(data.id,"BasicCommunication.SystemRequest", "SUCCESS", {})
+        end)
+      EXPECT_RESPONSE(CorIdSystemRequest, {success = true, resultCode = "SUCCESS"})
+      :Do(function(_,data)
+          --Copy of JSON file in /tmp/fs/mp/images/ivsu_cache/
+          --Example: os.execute("cp /home/anikolaev/OpenSDL_AUTOMATION/test_run/files/ptu.json /tmp/fs/mp/images/ivsu_cache/")
+          self.hmiConnection:SendNotification("SDL.OnReceivedPolicyUpdate",
+            {
+              policyfile = "/tmp/fs/mp/images/ivsu_cache/ptu.json"
+            })
+        end)
+      :Do(function(_,data)
+          EXPECT_HMINOTIFICATION("SDL.OnStatusUpdate", {status = "UP_TO_DATE"})
+        end)
+    end)
+end
+--[[ Preconditions ]]  
+commonFunctions:newTestCasesGroup("Preconditions")
+function Test:Pre_MoveSystemInUpToDateStatus()
+  policyUpdate(self, "/tmp/fs/mp/images/ivsu_cache/ptu.json")
+end
+ 
+
+function Test:Pre_OpenNewSession()
+  self.mobileSession2 = mobile_session.MobileSession(self, self.mobileConnection)
+  self.mobileSession2:StartService(7)
+end
+
+function Test:Pre_RAI_InNewSession()
+  local registerAppInterfaceParams =
+  {
+    syncMsgVersion =
+    {
+      majorVersion = 3,
+      minorVersion = 0
+    },
+    appName = "Media Application",
+    isMediaApplication = true,
+    languageDesired = 'EN-US',
+    hmiDisplayLanguageDesired = 'EN-US',
+    appHMIType = {"NAVIGATION"},
+    appID = "MyTestApp",
+    deviceInfo =
+    {
+      os = "Android",
+      carrier = "Megafon",
+      firmwareRev = "Name: Linux, Version: 3.4.0-perf",
+      osVersion = "4.4.2",
+      maxNumberRFCOMMPorts = 1
+    }
+  }
+
+  local corId = self.mobileSession2:SendRPC("RegisterAppInterface", registerAppInterfaceParams)
+  EXPECT_HMINOTIFICATION("BasicCommunication.OnAppRegistered", { application = { appName = "Media Application" }})
+  self.mobileSession2:ExpectResponse(corId, { success = true, resultCode = "SUCCESS" })
+  self.mobileSession2:ExpectNotification("OnPermissionsChange")
+end
+
+--[[ Test ]]
+commonFunctions:newTestCasesGroup("Test Case")
+--Copy prepared JSON file with ALLOWED RPC ListFiles, in /tmp/fs/mp/images/ivsu_cache/ - and use it.
+--Example: os.execute("cp /home/anikolaev/OpenSDL_AUTOMATION/test_run/files/PTU_NewPolicy_GENEVI.json /tmp/fs/mp/images/ivsu_cache/") 
+function Test:AssignNewDefaultPoicies()
+   policyUpdate(self, "/tmp/fs/mp/images/ivsu_cache/PTU_NewPolicy_GENEVI.json")
```

Comment:
@Aleksandriys , file PTU_NewPolicy_GENEVI.json is not committed - so I cannot check correctness of script bahavior